src/tests/Login.test.jsx

```diff
@@ -107,3 +106,23 @@ it('handles wrong credentials', async () => {
     
     expect(screen.getByLabelText('errorMessage')).toHaveTextContent("The server is currently unavailable. Try again later")
   })
+
+it('checks if the fields are empty', async () => {
+
+    const{getByLabelText} = render(<Login />)
```

Comment:
No need to declare `render ...` as constant. To be consistent with the other tests (and within this test) just use `screen.getByLabelText` as you did in lines 114-119.